src/SFA.DAS.ApplyService.Application/Apply/Roatp/UpdateApplicationStatusHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Roatp
+{
+    public class UpdateApplicationStatusHandler : RequestHandler<UpdateApplicationStatusRequest, bool>
+    {
+        private ILogger<UpdateApplicationStatusHandler> _logger;
+        private readonly IApplyRepository _repository;
+
+        public UpdateApplicationStatusHandler(ILogger<UpdateApplicationStatusHandler> logger, IApplyRepository repository)
+        {
+            _logger = logger;
+            _repository = repository;
+        }
+
+        protected override bool Handle(UpdateApplicationStatusRequest request)
+        {
+            _logger.LogInformation($"Updating application status to {request.ApplicationStatus} for application ID {request.ApplicationId}");
+
+            _repository.UpdateApplicationStatus(request.ApplicationId, request.ApplicationStatus);
```

Comment:
this is optional.... maybe wrap in a try catch error and only proceed to 'true' if it's all ok?

Reply:
I think it should log the failure and proceed, the key outcome is that the user is able to restart their application from scratch. Any issues with updating could be picked up as a data fix